runtime/oti/j9consts.h

```diff
@@ -174,7 +174,7 @@ extern "C" {
 #define J9_PRIVATE_FLAGS_STACKS_OUT_OF_SYNC 0x40000000
 #define J9_PRIVATE_FLAGS_FINAL_CALL_OUT_OF_MEMORY 0x80000000
 
-#define J9_PRIVATE_FLAGS2_UNUSED_0x1 0x1
+#define J9_PRIVATE_FLAGS2_IN_SHARED_CLASS_READ_MUTEX 0x1
```

Comment:
I think CACHE instead of CLASS is more accurate.

Reply:
Fixed.